src/caffe/proto/caffe.proto

```diff
@@ -1184,6 +1193,7 @@ message V1LayerParameter {
     SPLIT = 22;
     SLICE = 33;
     TANH = 23;
+    TRIPLET_LOSS = 40;
```

Comment:
In my opinion this isn't needed since the `TripletLossLayer` was not in Caffe while the V1 protobuf were in use. @jeffdonahue @longjon should we have some kind of backward compatibility in this case?